ROADMAP.md

```diff
@@ -158,4 +157,79 @@ The following features are under design review:
+
+* Katib 0.10 with the new v1beta1 API 
+* Katib support for early stopping.
+* Katib support for custom CRD in the new Trial template.
+* Katib support to resume experiments
+* Katib support for multiple ways to extract metrics
+* KFServing support to add batcher module as sidecar 
+* KFServing for the Alibi explainer upgrade to 0.4.0 
+* KFServing for Triton inference server rename and integrations  
+* Pipelines support for a Tekton backend option.
+* Kubeflow Pipelines 1.0.4, Changelog includes ~20 fixes and ~5 minor features.
+* Notebooks support for Affinity/Toleration configs 
+* Update mxnet-operator manifest to v1
+* Correct XGBoostJob CRD group name and add singular name
+* Fix XGBoost Operator manifest issue
+* Move Pytorch operator e2e tests to AWS Prow
+* Support BytePS in MXNet Operator
+* Fix error when conditions is empty in tf-operator
+* Fix success Policy logic in MXNet Operator
+
+For more details please see this post: https://blog.kubeflow.org/release/official/2020/11/18/kubeflow-1.2-blog-post.html
+
+## Kubeflow 1.3 Features, Target release: March 2021
+
+The Kubeflow 1.3 roadmap includes many User Interface (UI) improvements and core Kubeflow component upgrades to improve installation, management, and authentication.  It also includes support the latest Istio versions.
+
+The 1.3 release plan includes the following features:
+
+User Interface (UI) & Working Group enhancements to improve user experience and simplify workflows & operations
+
+* Completely new UIs for KFServing, Katib, Tensorboard & Volumes Manager
+* Notebooks
+  * Important backend updates to Notebooks (i.e. to improve interop with Tensorboard)
+  * New and expanded Jupyter Notebook stack along with easy to customize common base images - this is a stretch feature for 1.3
+  * Addition of R-Studio and Code-Server (VS-Code) support
+* Kubeflow Pipelines (KFP) 
+  * UI reorganization for better User Experience
+  * Manage recurring Runs via new “Jobs” page (exact name on UI is TBD)
+  * Simplified view of dependency graphs 
+  * Multi-user feature enhancements in Kubeflow Pipelines
+* KFServing v0.5
+   * Ability to specify container fields on ML Framework spec such as env variable, liveness/readiness probes etc.
+   * Ability to specify pod template fields on component spec such as NodeAffinity etc.
+   * gRPC support Tensorflow Serving.
+   * Triton Inference server V2 inference REST/gRPC protocol support
+   * TorchServe predict integration
+   * PyTorch Captum explain integration
+   * SKLearn/XGBoost V2 inference REST/gRPC protocol support with MLServer
+   * PMMLServer support
+   * LightGBM support
+   * Allow specifying timeouts on component spec
+   * Simplified canary rollout, traffic split at knative revisions level instead of services level
+   * Transformer to predictor call is now made async
+
+Core improvements to Kubeflow Installation, Management, Authentication, and Istio
+
+* Support for latest Istio versions across Kubeflow applications:
```

Comment:
I think KFServing should another application which support latest istio version, we should either 

1) Communicate with KFServing WG and confirm 

or 

2) Wait for KFServing WG response (this may be late), otherwise, there might be a gap between different KF applications supported istio version

Reply:
@PatrickXYS Hey Patrick, the Kubeflow 1.3 Release Management team confirmed their testing of KFServing 0.5 with Istio 1.9.   This is in the notes of the manifest working group meeting, link below. https://docs.google.com/document/d/1d-QByp_QRMkxRaQER53mYfeRW0-KyhDBPdmeMEzkL0g/edit

Reply:
If so, can we add it in the doc as well?

Reply:
@PatrickXYS I want to make sure that I understand your request...Would you like the roadmap to reference the manifest working group notes on the Istio roadmap line item ?

Reply:
Hey @jbottum , I'm suggesting you add another sub-bullet point under `* Support for latest Istio versions across Kubeflow applications:` for KFServing just like the way you did for KFP.

E.g, 

```
* Support for latest Istio versions across Kubeflow applications:
  * KFP, Profile-Controller and KFAM will support the new AuthorizationPolicy API
  * KFSering 0.5 is compatible with the latest istio version
```

Reply:
But that's only nit suggestions.

/lgtm